plugins/auto-sizes/includes/improve-calculate-sizes.php

```diff
@@ -255,15 +263,15 @@ function auto_sizes_filter_uses_context( array $uses_context, WP_Block_Type $blo
 	return $uses_context;
 }
 
-/**
- * Modifies the block context during rendering to blocks.
- *
- * @since n.e.x.t
- *
- * @param array<string, mixed> $context Current block context.
- * @param array<string, mixed> $block   The block being rendered.
- * @return array<string, mixed> Modified block context.
- */
+	/**
+	 * Modifies the block context during rendering to blocks.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param array<string, mixed> $context Current block context.
+	 * @param array<string, mixed> $block   The block being rendered.
+	 * @return array<string, mixed> Modified block context.
+	 */
```

Comment:
```suggestion
/**
 * Modifies the block context during rendering to blocks.
 *
 * @since n.e.x.t
 *
 * @param array<string, mixed> $context Current block context.
 * @param array<string, mixed> $block   The block being rendered.
 * @return array<string, mixed> Modified block context.
 */
```